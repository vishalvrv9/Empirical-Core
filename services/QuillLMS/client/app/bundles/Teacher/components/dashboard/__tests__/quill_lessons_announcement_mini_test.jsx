import React from 'react';
import { shallow } from 'enzyme';

import QuillLessonsAnnouncementMini from '../quill_lessons_announcement_mini';

describe('QuillLessonsAnnouncementMini component', () => {

  it('should render', () => {
    expect(shallow(<QuillLessonsAnnouncementMini />)).toMatchSnapshot();
  });

});
