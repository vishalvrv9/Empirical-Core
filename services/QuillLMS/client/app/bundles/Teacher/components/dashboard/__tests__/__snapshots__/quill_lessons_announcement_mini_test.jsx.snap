// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuillLessonsAnnouncementMini component should render 1`] = `
<div
  className="mini_container results-overview-mini-container col-md-4 col-sm-5 text-center"
>
  <div
    className="mini_content"
  >
    <div
      className="gray-underline"
      style={
        Object {
          "padding": "8px",
          "position": "relative",
        }
      }
    >
      <img
        src="https://assets.quill.org/images/icons/icon-star.svg"
        style={
          Object {
            "height": "27px",
            "position": "absolute",
            "right": "-22px",
            "top": "-11px",
            "transform": "rotate(-22deg)",
            "width": "27px",
          }
        }
      />
      <img
        src="https://assets.quill.org/images/icons/icon-star.svg"
        style={
          Object {
            "height": "14px",
            "position": "absolute",
            "right": "-3px",
            "top": "-11px",
            "transform": "rotate(-34deg)",
            "width": "14px",
          }
        }
      />
      <h3
        style={
          Object {
            "fontWeight": "normal",
          }
        }
      >
        New In-Class Collaborative Tool
      </h3>
    </div>
    <div>
      <a
        href="/tools/lessons"
        style={
          Object {
            "alignItems": "center",
            "display": "flex",
            "justifyContent": "center",
            "marginTop": "8px",
          }
        }
      >
        <img
          src="/images/lesson_icons_black.svg"
          style={
            Object {
              "height": "25px",
              "marginRight": "10px",
              "marginTop": "3px",
            }
          }
        />
        <p
          style={
            Object {
              "fontSize": "18px",
              "fontWeight": "bold",
              "margin": "8px 0 0",
            }
          }
        >
          Quill Lessons
        </p>
      </a>
    </div>
    <p
      style={
        Object {
          "fontFamily": "lucida-grande, adelle-sans, helvetica",
          "lineHeight": "1.57",
          "marginTop": "16px",
          "minHeight": "86px",
          "padding": "0px 15px",
        }
      }
    >
      Use Quill Lessons to lead whole-class direct instruction with interactive lesson plans. With Quill Lessons, you can see and display student responses in real time.
    </p>
    <a
      href="https://medium.com/writing-with-quill/quill-launches-quill-lessons-fb983f2b8970"
      target="_blank"
    >
      <button
        className="button button-white beta"
        style={
          Object {
            "fontWeight": "normal",
            "maxWidth": "233px",
            "paddingBottom": "13px",
            "width": "100%",
          }
        }
      >
        Learn More
      </button>
    </a>
  </div>
</div>
`;
